From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aditya@kobol.io>
Date: Tue, 15 Sep 2020 20:04:22 +0700
Subject: arm64: dts: rockchip: helios64: name the fans after their headers

fan1 -> p7-fan, fan2 -> p6-fan, with labels; 90-helios64-hwmon.rules
matches KERNELS=="p6-fan"/"p7-fan".

Upstream: none (Armbian naming).

Split out of add-board-helios64.patch (Aditya Prayoga, rebased onto
mainline by rpardini for 6.3); the DTB content is unchanged.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
Assisted-by: LLM
Signed-off-by: Igor Velkov <iav@iav.lv>
---
 arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-kobol-helios64.dts
@@ -74,14 +74,14 @@ / {
 		#clock-cells = <0>;
 	};
 
-	fan1 {
+	fan1: p7-fan {
 		/* fan connected to P7 */
 		compatible = "pwm-fan";
 		pwms = <&pwm0 0 40000 0>;
 		cooling-levels = <0 80 170 255>;
 	};
 
-	fan2 {
+	fan2: p6-fan {
 		/* fan connected to P6 */
 		compatible = "pwm-fan";
 		pwms = <&pwm1 0 40000 0>;
-- 
Armbian

